docs/docs.go

```diff
@@ -2,3 +2,537 @@
 // This file was generated by swaggo/swag
 
 package docs
+
```

Comment:
get the `.gitignore` and `placeholder.go` from master and do not upload new docs.go anymore. My bad on this,

Reply:
Emmm...did you mean delete the docs.go file?

Reply:
Yes